rpc/src/rpc.rs

```diff
@@ -348,7 +348,7 @@ impl JsonRpcRequestProcessor {
         )
     }
 
-    // Useful for unit testing
+    #[cfg(test)]
```

Comment:
nit: We do have some precedent for ending function names like this with `_for_tests`. Ie,
```rust
pub fn new_from_bank_for_tests(
```
That being said, `#[cfg(test)]` makes it functionally true that this is for tests whereas the name is more of a "suggestion" so I'd be ok pushing things as-is